waspc/data/Generator/templates/sdk/wasp/client/operations/queries/core.ts

```diff
@@ -1,11 +1,16 @@
+import { Route } from 'wasp/client'
+import type { Expand, _Awaited, _ReturnType } from 'wasp/universal/types'
+import { type Query } from '../core.js'
 import { callOperation, makeOperationRoute } from '../internal/index.js'
 import {
   addResourcesUsedByQuery,
   getActiveOptimisticUpdates,
 } from '../internal/resources'
 
-// PRIVATE API
-export function createQuery(relativeQueryPath, entitiesUsed) {
+export function createQuery<BackendQuery extends GenericBackendQuery>(
```

Comment:
Declaration merged from `core.d.ts`. Same goes for `queries/core.ts`.

I had to do this because our `core.d.ts` file would get overwritten during compilation - TypeScript would compile the `core.js` file and take the types from there. These types were useless - all of them were `any`.